README.md

```diff
@@ -426,6 +427,13 @@ If a user agent partitions service workers using this scheme, there is no cross-
 
 [Service workers are disabled](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Privacy/State_Partitioning) in Firefox when Dynamic Partitioning is enabled, but they are [working on implementing a partitioned service worker solution](https://bugzilla.mozilla.org/show_bug.cgi?id=1495241).
 
+## Security and Considerations
+
+### Security
```

Comment:
Given that the section is so short, I wouldn't bother breaking it into security and privacy subsections. If you were to break it up, I think the more common convention is to have separate "Security Considerations" and "Privacy Considerations" top-level sections.

Reply:
Whoops, mean to delete that line. Fixed.